src/Renci.SshNet/.editorconfig

```
@@ -1,4 +1,7 @@
 ﻿[*.cs]
+spelling_languages = en-us,en-gb
+spelling_exclusion_path = ../exclusion.dic
```

the file this is pointing to has only 2 words, both already included in the top-level file. Is it an artifact of earlier changes?

```suggestion
spelling_exclusion_path = ../../exclusion.dic
```